rs/nns/governance/src/neuron/types.rs

```diff
@@ -318,7 +318,9 @@ impl Neuron {
         // Main calculation.
         let result = adjustment_factor * Decimal::from(potential_voting_power);
 
-        // Convert (back) to u64.
+        // Convert (back) to u64. Currently, Decimal uses "banker's rounding".
```

Comment:
Note that banker's rounding is the default strategy, but the library supports explicit rounding strategy specification, 

e.g., we do that [here](https://sourcegraph.com/github.com/dfinity/ic/-/blob/rs/nervous_system/neurons_fund/src/lib.rs?L55).

Reply:
The small benefit is that, if the library's next version changes its mind on the default rounding, we'd still have the same logic.

NCR as I don't think this is a practical risk for us here.

Reply:
Done.